models/src_iterable.yml

```diff
@@ -65,9 +68,15 @@ sources:
             description: Last update timestamp as epoch time in milliseconds
 
       - name: campaign_suppression_list_history  # this is the correct spelling
+        # if statements in identifier are to account for historical variables used for this table while also allowing new format to be used
+        identifier:  >
+          {%- if var('iterable__campaign_suppression_list_history_table', False) -%}
```

Comment:
The False here is the default value of the var. If `iterable__campaign_suppression_list_history_table` exists, it will return True to the if statement, but if not then it will be false. I went this route, so it would go on to checking for a `iterable_campaign_suppression_list_history_identifier`. Though now that you mention it, I might ping the team if there is a more efficient way to do this.

Reply:
took out this logic per Joe's comment yesterday.